Library/Homebrew/extend/ENV/std.rb

```diff
@@ -132,7 +132,7 @@ def determine_cc
   # @private
   def determine_cxx
     dir, base = determine_cc.split
-    dir / base.to_s.sub("gcc", "g++").sub("clang", "clang++")
+    dir / base.to_s.sub("gcc", "g++").sub("clang", "clang++").sub(/\bcc\b/, "c++")
```

Comment:
Can you please elaborate why this change was needed? What command did you run, and what error message did you see? What was the value of `CC` and what was the resulting value of `CXX`?